.rubocop.yml

```diff
@@ -82,6 +86,12 @@ Metrics/ParameterLists:
   Max: 4
 Metrics/PerceivedComplexity:
   Max: 8
+Naming/FileName:
+  Enabled: false
+Naming/HeredocDelimiterCase:
+  Enabled: false
```

Comment:
We should enable this!

Reply:
okay.. I'll go with enforcing `uppercase` (default)